.github/workflows/release.yml

```diff
@@ -60,23 +60,26 @@ jobs:
     environment: release
     runs-on: ubuntu-22.04
     needs: pypi
+    if: github.event_name == 'release'
 
     env:
       FORCE_COLOR: 1
       PY_COLORS: 1
       TOXENV: pkg
 
     steps:
-      - name: Check out src from Git
-        uses: actions/checkout@v4
```

Comment:
Checking out the source for this repository is not necessary to run `bump-homebrew-formula`.